supportedBackends/google/pipelines/batch/src/test/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfigurationAttributesSpec.scala

```diff
@@ -0,0 +1,471 @@
+package cromwell.backend.google.pipelines.batch
+
+import cats.data.Validated.{Invalid, Valid}
+import cats.syntax.validated._
+import com.typesafe.config.{Config, ConfigFactory}
+import common.assertion.CromwellTimeoutSpec
+import common.exception.MessageAggregation
+import cromwell.backend.google.pipelines.batch.GcpBatchTestConfig.BatchGlobalConfig
+import cromwell.backend.google.pipelines.batch.GcpBatchConfigurationAttributes._
+import cromwell.cloudsupport.gcp.GoogleConfiguration
+import cromwell.cloudsupport.gcp.auth.MockAuthMode
+import cromwell.filesystems.gcs.GcsPathBuilder
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+//import java.net.URL
+import scala.concurrent.duration._
+
+// TODO: Need to replace CromwellTimeoutSpec with appropriate class from batch
+class GcpBatchConfigurationAttributesSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers
+  with TableDrivenPropertyChecks {
+
+  behavior of "PipelinesApiAttributes"
```

Comment:
Change wording to Batch

Reply:
Wording has been changed to 'GcpBatchAttributes'